.eslintrc.js

```diff
@@ -43,10 +39,6 @@ module.exports = {
   },
 
   overrides: [
-    {
-      files: ['*.{js,jsx}'],
-      parser: 'babel-eslint',
-    },
```

Comment:
After dependency update there were some errors because of this. We do not have any js/jsx files in repo, therefore I've taken the liberty of removing this override.